integration-tests/tests/fixtures/package_config.py

```diff
@@ -0,0 +1,22 @@
+from pathlib import Path
+
+import pytest
+from tests.utils.config import PackageConfig
+from tests.utils.utils import get_env_var
+
+
+@pytest.fixture(scope="session")
+def package_config() -> PackageConfig:
+    clp_build_dir = Path(get_env_var("CLP_BUILD_DIR")).resolve()
+    clp_package_dir = Path(get_env_var("CLP_PACKAGE_DIR")).resolve()
```

Comment:
I still feel that we should error out here if  clp_package_dir doesn't exist, instead of delaying it till later